src/System.Net.Sockets/src/System/Net/Sockets/IOControlKeepAlive.cs

```diff
@@ -0,0 +1,93 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+
+namespace System.Net.Sockets
+{
+    internal class IOControlKeepAlive
+    {
+        private const uint OnOff = 1u;
+        private const uint DefaultTimeMs = 7200000u;
+        private const uint DefaultIntervalMs = 1000u;
+        private static readonly ConditionalWeakTable<Socket, IOControlKeepAlive> SocketKeepAliveCwt;
+        private static readonly int UIntSize;
```

Comment:
There's no need to store this in a field.  You can just use `sizeof(uint)` wherever you need it.